### ISSUE TITLE
Games not able to dynamically add input (xbox360 wireless) during gameplay

### ISSUE THREAD

**Issue by AsavarTzeth** [ValveSoftware/SteamOS#205]
I am not sure if this a SteamOS/xpad issue or a case of game devs doing something wrong. But what I do know is that almost all games have to be restarted if you are adding a wireless Xbox360 controller while playing.

Currently this is mostly an issue because the gamepad will turn itself off to save battery if you leave it for a while. Recently when playing BattleBlock Theater this is an even greater issue because the game cannot fall back to using keyboard and mouse, leaving me stranded and having to kill the game or rebooting the PC.

From an end-user perspective I see this as a SteamOS killer bug. Of course, I have not tested this on other controllers recently so it might just be me having issues (I doubt that though).

There are a few games that does not have this issue, I think. I do not remember which ones fot sure though and my SteamOS install just crashed and randomly broke completely so I cannot test atm.

System Information:
Intel i7 2600K
Gigabute GTX 660 Ti
Steam Beta

**Comment by ProfessorKaos64** [comment]
This is not an OS problem. Many games employ different input paradigms, such as xinput, via SDL2, and so on. I double you will see any traction on this. Case in point, is the controller configurations in Steam are handled via SDL2. How would Valve force the game developer to to use it? It is not up to Valve to make this change.

**Comment by AsavarTzeth** [comment]
I already know this. Since originally this I am well aware of how input is handled.

If my memory serves me I created this issue both under SteamOS and Steam issue tracker, because I was unsure where it belonged. I then got an answer from a collaborator well over a year ago on the other issue tracker. It would seem I forgot this entry.

If you read the whole issue description you will see the issue is really about losing input control during gameplay. The situation today is dramatically different. Now we have the finished Steam Controller and since that is the officially recommenced and bundled gamepad my main issue is in a way solved.

It is relevant to remember though that when I posted this it was from an end-user perspective and most regular end-users do not care who is responsible.

Closing
